docs/use/toolchain.md

```diff
@@ -0,0 +1,81 @@
+# Lind toolchain
+
+The toolchain to build and run lind programs consists of the following components:
+
+- *Clang* with *WebAssembly System Interface* (WASI) support to build *glibc*
+  and lind programs
+- A custom [*glibc*](../internal/libc.md) used as sysroot to build
+  lind programs
+- A custom `wasm-opt` binary to enable [multi-processing](../internal/multiprocess-support.md)
```

Comment:
is the method of customization of either this or the following custom WebAssembly explained anywhere? Is it important to?

Reply:
Good point! I think the `internal/*` documentation should and in parts already does elaborate on this.

Reply:
Maybe we can link to that in a footnote or something on this and expand on it in the appropriate place

Reply:
I think there's a bit more to this to do with signaling, hopefully @qianxichen233 can clarify.

Reply:
> What do you think about moving the link to wasm-opt and create an issue for mentioning it in the mutli-processing doc?

Just changed the link, will post an issue shortly.